plugins/lyrics-genius/back.js

```diff
@@ -38,7 +64,11 @@ const fetchFromGenius = async (metadata) => {
 	}
 
 	if (is.dev()) {
-		console.log("Fetching lyrics from Genius:", url);
+		if(revRomanized) {
+			console.log("Fetching romanized lyrics from Genius:", url);
+		} else {
+			console.log("Fetching lyrics from Genius:", url);
+		}
```

Comment:
This is not a very useful log since it's just checking if an option is toggled

I'd say revert the log to how it previously was

```suggestion
		console.log("Fetching lyrics from Genius:", url);
```

Reply:
Got it